tests/integration/change_detector.go

```diff
@@ -64,10 +70,21 @@ func detectDbChangesInit(repository string, targetBranch string) {
 		return
 	}
 
-	tempDir := os.TempDir()
+	defraTempDir := path.Join(os.TempDir(), "defradb")
+	changeDetectorTempDir := path.Join(defraTempDir, "tests", "changeDetector")
 
 	latestTargetCommitHash := getLatestCommit(repository, targetBranch)
-	detectDbChangesCodeDir = path.Join(tempDir, "defra", latestTargetCommitHash, "code")
+	detectDbChangesCodeDir = path.Join(changeDetectorTempDir, "code", latestTargetCommitHash)
+	rand.Seed(time.Now().Unix())
+	randNumber := rand.Int()
+	dbsDir := path.Join(changeDetectorTempDir, "dbs", fmt.Sprint(randNumber))
+
+	testPackagePath, isIntegrationTest := getTestPackagePath()
+	if !isIntegrationTest {
```

Comment:
question: When would that not be an integration test?

Reply:
The bench suite uses this, and there might be other references too.